addons/website_slides_survey/models/res_partner.py

```diff
@@ -0,0 +1,31 @@
+# -*- coding: utf-8 -*-
+# Part of Odoo. See LICENSE file for full copyright and licensing details.
+
+from odoo import api, fields, models
+
+
+class ResPartner(models.Model):
+    _inherit = 'res.partner'
+
+    certification_ids = fields.One2many('slide.slide.partner', 'partner_id', domain=[('survey_quizz_passed', '=', True)])
+    certifications_count = fields.Integer('Succeeded Certifications Count', compute='_compute_certifications_count')
+    company_certification_ids = fields.One2many('slide.slide.partner', 'partner_id', compute='_compute_company_certifications_count')
+    company_certifications_count = fields.Integer('Succeeded Certification By Company Partners', compute='_compute_company_certifications_count')
+
+    def _compute_certifications_count(self):
+        self.certifications_count = len(self.certification_ids)
```

Comment:
read_group_res = self.env['slide.slide.partner'].sudo().read_group(
  [('partner_id', 'in', self.ids), ('survey_quizz_passed', '=', True)],
  ['partner_id'],
  'partner_id')

See for example _compute_members_count in slide_channel.py